packages/elements/src/slider/elements/slider.ts

```diff
@@ -897,11 +897,16 @@ export class Slider extends FormFieldElement {
       return;
     }
 
-    const { value, name } = event.target as NumberField;
+    const inputEl = event.target as NumberField;
+    const { value, name } = inputEl;
 
     const currentData = name as SliderDataName;
     const previousData = `${name}Previous` as SliderPreviousDataName;
 
+    if (!value) {
+      inputEl.value = this[currentData];
+    }
+
```

Comment:
Suggesting a minor improvement kub. I think this improves a bit of overall code readability.

```ts
    if (value) {
      if (this[currentData] !== value) {
        this.updateNotifyProperty(currentData, value);
        this[previousData] = value;
      }
    } else {
      inputEl.value = this[currentData];
    }

```

Reply:
if we were to refactor this part I would suggest:
```typescript
    if (!value) {
      inputEl.value = this[currentData];
    } else if (this[currentData] !== value) {
      this.updateNotifyProperty(currentData, value);
      this[previousData] = value;
    }
```

Reply:
Either way is definitely an improvement kub. I just prefer starting the statement with positivity if it's not a guard clause.